vcu/Src/drive_by_wire_mock.c

```diff
@@ -440,6 +440,75 @@ static const CLI_Command_Definition_t mcInitCommandDefinition =
     0 /* Number of parameters */
 };
 
+
+BaseType_t setInverterParameter(char *writeBuffer, size_t writeBufferLength, const char *commandString){
```

Comment:
to turn on and off the motor controllers include the definitions of `turnOffMotorControllers` and `turnOnMotorControllers` in drive_by_wire.h and call the functions in your CLI command